package.json

```
@@ -34,6 +34,7 @@
     "clean": "turbo run clean --filter=!pie-monorepo",
     "cz": "./packages/tools/pie-git-hooks-scripts/check-branch-name.js $(git symbolic-ref --short HEAD) && ./node_modules/cz-customizable/standalone.js",
     "dev": "turbo run dev --filter=!pie-monorepo",
+    "copy:component-statuses": "cross-env-shell turbo run copy:component-statuses --filter=!pie-monorepo --token=${TURBO_TOKEN}",
```

Since caching isn't enabled for this task.

```suggestion
    "copy:component-statuses": "cross-env-shell turbo run copy:component-statuses --filter=!pie-monorepo",
```